lib/k8s-e2e-tests/tests/vector-agent.rs

```diff
@@ -11,6 +11,9 @@ use tracing::{debug, info};
 const HELM_CHART_VECTOR_AGENT: &str = "vector-agent";
 
 const HELM_VALUES_STDOUT_SINK: &str = indoc! {r#"
+    kubernetesLogsSource:
+      rawConfig: |
+        glob_minimum_cooldown_ms = 5000
```

Comment:
```suggestion
      config:
        glob_minimum_cooldown_ms: 5000
```

Reply:
I was going to ask you about this. That suggested change gives me this error within Vectors logs:

```
Jun 03 17:18:48.037  INFO vector::app: Loading configs. path=[("/etc/vector/managed.toml", None)]
Jun 03 17:18:48.052 ERROR vector::cli: Configuration error. error=invalid type: floating point `5000`, expected usize for key `sources.kubernetes_logs` at line 45 column 1
```

Any ideas why?

I'm experimentally making that change largely to see if CI passes. If it does, I think it will be worth making 5000 the default so I won't need that config change in the test.

Reply:
_Ugh_ that would be this I suspect: #7453

In that case I can just remove it with the rest of the deprecation work

Reply:
That floating point error is due to #7453 . I know @spencergilbert has been poking at it. I'm hoping we can fix it for 0.15.0.